tool/mapeditor.re

```diff
@@ -1,18 +1,32 @@
 static mut MAP_EDITOR_STATE = MapEditorState {
     map_name: "",
     map: Tas::current_map(),
+    mode: MapMode::Play,
 };
 
 struct MapEditorState {
     map_name: string,
     map: RefunctMap,
+    mode: MapMode,
+}
+
+enum MapMode {
```

Comment:
As all identifiers in rebo are currently global, changing the name should make it clearer where it belongs to (e.g. maybe in the future we want to add different playing modes to maps themselves and call them `MapMode`).
```suggestion
enum MapEditorMode {
```